scripts/flashida/write_method.py

```
@@ -0,0 +1,105 @@
+import json
+import sys
+import pyopenms as oms
```

Flake8 (7.2.0)

[error] 'pyopenms as oms' imported but unused

(F401)